platform/android/plugin/godot_plugin_config.h

```diff
@@ -35,22 +35,22 @@
 #include "core/io/config_file.h"
 #include "core/string/ustring.h"
 
-static const char *PLUGIN_CONFIG_EXT = ".gdap";
+static const char *ANDROID_PLUGIN_CONFIG_EXT = ".gdap";
```

Comment:
Maybe those should be class constants so that they're properly namespaced? Then we don't need the suffix.